src/components/cardlist.jsx

```diff
@@ -0,0 +1,28 @@
+// eslint-disable jsx-props-no-spreading
+import CardItem from './carditem';
+
+const cardStyle = {
+  display: 'grid',
+  gridTemplateColumns: 'repeat(5, 1fr)',
+  gridGap: '1rem',
+  justifyItems: 'center',
+};
```

Comment:
Be consistent on the styling method on your whole project

Reply:
done